intro and arch (#121)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_introduction-to-stf.adoc

[id="introduction-to-stf_{context}"]
= Introduction to {Project}

{Project} ({ProjectShort}) provides automated collection of measurements and data from remote clients - {OpenStack} or third-party nodes - and transmission of that information to a centralized, receiving {OpenShift} ({OpenShiftShort}) deployment for storage, retrieval, and monitoring. The data can be either of two types:

* Metric - a numeric measurement of an application or system
* Event - irregular and discrete occurrences that happen in a system

> The collection components required on the clients are lightweight, and a multicast message bus, which is shared by all clients and the deployment, provides fast and reliable data transport. Other modular components for receiving and storing data are deployed in containers on {OpenShiftShort}.

Reviewer: the action 'are lightweight' refers to 'the collection of components'  collection is singular, so should that not be 'is lightweight' ? 
Upon rereading I think what you want to say is: 
```suggestion
The collection components that are required on the clients are lightweight. The multicast message bus that is shared by all clients and the deployment provides fast and reliable data transport. Other modular components for receiving and storing data are deployed in containers on {OpenShiftShort}.
```
